test/parallel/test-child-process-spawn-typeerror.js

```diff
@@ -58,32 +60,32 @@ assert.doesNotThrow(function() {
 // verify that invalid argument combinations throw
 assert.throws(function() {
   spawn();
-}, invalidFileMsg);
+}, invalidOptValueError);
 
 assert.throws(function() {
   spawn('');
-}, invalidFileMsg);
+}, invalidOptValueError);
 
 assert.throws(function() {
-  const file = { toString() { throw new Error('foo'); } };
+  const file = { toString() { return null; } };
```

Comment:
Is this intentional? What happens now if the error is thrown like in the old test?